pkg/planner/core/planbuilder.go

```diff
@@ -2684,7 +2684,7 @@ var analyzeOptionDefault = map[ast.AnalyzeOptionType]uint64{
 
 var analyzeOptionDefaultV2 = map[ast.AnalyzeOptionType]uint64{
 	ast.AnalyzeOptNumBuckets:    256,
-	ast.AnalyzeOptNumTopN:       500,
```

Comment:
Could you please explain why we changed the TopNNum from 500 to 100 in comments ?

Reply:
Added comment as requested